website/src/pages/product/code-learning.tsx

```diff
@@ -0,0 +1,212 @@
+                                structure, to track down where code lives, and self-service during their investigations.
+                            </p>
+                            <footer className="blockquote-footer">Owen Kim, Senior Software Engineer, Convoy</footer>
+                        </blockquote>
+                    </div>
+                    <div className="col-lg-10 mt-2 py-4 text-center">
+                        <h5>
+                            Learn more about how{' '}
+                            <a href="/blog/convoy-improved-their-developer-on-boarding-with-sourcegraph">
+                                Convoy uses Sourcegraph to onboard new developers.
+                            </a>
+                        </h5>
+                    </div>
+                </div>
+            </ContentSection>
+            <ContentSection color="white" className="pb-2">
+                <h2 id="features" className="display-4 mb-5 text-center">
+                    Why learning code is faster with Sourcegraph
+                </h2>
+                <p className="text-center mb-5">
+                    Sourcegraph <a href="/universal-code-search">Universal Code Search</a> lets you explore across your
+                    code and public codebases with contextual code intelligence to accelerate understanding code when
+                    onboarding new developers and reviewing unfamiliar code.
+                </p>
+                <div className="row">
+                    <div className="col-lg-6 mb-4">
+                        <IconItem className="position-relative" icon={GlobeIcon} color="green">
+                            <h4>Find definitions and explore references, even across projects</h4>
+                            <p>
+                                See a call to a function you don't recognize? With Sourcegraph, you can see its
+                                documentation and type signature by hovering over it &mdash; and go to its definition in
+                                a single click, across package, dependency, and repository boundaries. Even if your IDE
+                                can do this, Sourcegraph lets you explore code intelligently in your web browser in any
+                                repository on any branch, instantly and without losing your local context to massively
+                                save time. Works for <SupportedProgrammingLanguagesLink />.
+                            </p>
+                        </IconItem>
+                    </div>
+                    <div className="col-lg-6 mb-4">
+                        <IconItem className="position-relative" icon={PlaylistAddCheckIcon} color="brand-cyan">
+                            <h4>
+                                Facilitate easier onboarding, with rapidly growing and distributed teams and codebases
```

Comment:
"Facilitate" is a throwaway word. Can we just remove it? "Onboard faster"?

"Distributed teams" is mentioned, but we don't explain how that applies. The body text below explains how Sourcegraph helps you with big codebases, but it doesn't explain how it helps when your team is distributed. We could say something about how when you can't tap people on the shoulder, you need a way to find answers on your own. But that's jamming too many ideas into a single paragraph IMO.

Also "distributed" adjective seems to apply to "codebases", which does not make sense.